test/common/common.go

```diff
@@ -138,6 +138,11 @@ func RemoveExtension(shoot *gardencorev1beta1.Shoot, extensionType string) {
 	})
 }
 
+// IsVerticalPodAutoscalerEnabled returns true if the VPA is enabled
+func IsVerticalPodAutoscalerEnabled(shoot *gardencorev1beta1.Shoot) bool {
```

Comment:
There is already such a helper func in gardener/gardener: [`ShootWantsVerticalPodAutoscaler`](https://github.com/gardener/gardener/blob/ee8b992d9ad818c3f8ad5d4a04c67fc4b157a316/pkg/apis/core/v1beta1/helper/shoot.go#L40-L43)